cpp/src/io/json/json_tree.cu

```diff
@@ -439,210 +457,227 @@ rmm::device_uvector<size_type> hash_node_type_with_field_name(device_span<Symbol
 }
 
 /**
- * @brief Translates sorted parent_node_ids to parent_indices with indices from scatter_indices
- *
- * @param scatter_indices The sorted order of parent_node_ids
- * @param parent_node_ids The sorted parent_node_ids
- * @param stream CUDA stream used for device memory operations and kernel launches
- * @return Translated parent_indices pointing to sorted node_ids positions
- */
-rmm::device_uvector<NodeIndexT> translate_sorted_parent_node_indices(
-  device_span<size_type const> scatter_indices,
+ * @note Two level hashing algorithm
```

Comment:
Humnn, if not doxygen then this should probably rewritten using the regular comment style:
```
// ...
// ...
// ...
```